frontend/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:18-alpine
+
+WORKDIR /app
+
+COPY package.json yarn.lock .
+
+RUN yarn install --frozen-lockfile
+
+COPY . .
+
+RUN yarn build
+CMD yarn serve dist -l 0.0.0.0:3000
+
+EXPOSE 3000
```

Comment:
It would be better if we can add entrypoint at the bottom of dockerfile so that we can run frontend app after launch container

Reply:
`CMD` and `ENTRYPOINT` serve the same purpose